### ISSUE TITLE
Go/fasthttp can't find package github.com/valyala/tcplisten

### ISSUE THREAD

**Issue by knewmanTE** [TechEmpower/FrameworkBenchmarks#2287]
All of the Go/fasthttp tests are failing because they can't find the package github.com/valyala/tcplisten. See test run output below:
```
Setup fasthttp-postgresql-prefork: qtc: 2016/09/21 06:55:26 Compiling *.qtpl template files in directory "."

Setup fasthttp-postgresql-prefork: qtc: 2016/09/21 06:55:26 Compiling "fortune.qtpl" to 
"fortune.qtpl.go"...

Setup fasthttp-postgresql-prefork: qtc: 2016/09/21 06:55:26 Total files compiled: 1

Setup fasthttp-postgresql-prefork: src/github.com/valyala/fasthttp/reuseport/reuseport.go:13:2: cannot 
find package "github.com/valyala/tcplisten" in any of:

Setup fasthttp-postgresql-prefork: 	/home/travis/build/TechEmpower/FrameworkBenchmarks
/installs/go/src/github.com/valyala/tcplisten (from $GOROOT)

Setup fasthttp-postgresql-prefork: 	/home/travis/build/TechEmpower/FrameworkBenchmarks
/frameworks/Go/fasthttp/src/github.com/valyala/tcplisten (from $GOPATH)

fasthttp-postgresql-prefork: setup-postgresql-prefork.sh and framework processes have terminated

Setup fasthttp-postgresql-prefork: Setup has terminated
```

ping @valyala 

For reference, I'm seeing this on the Travis run for #2286 and am doing more thorough local testing, so it's possible this isn't a real concern.

**Comment by knewmanTE** [comment]
Closing this issue because I believe the issue might be with that PR and not with the current fasthttp implementation itself.

**Comment by valyala** [comment]
This should be fixed by #2271 .
